packages/messages/src/browser/style/notifications.css

```diff
@@ -191,17 +191,27 @@
     margin-left: 4px;
 }
 
+.theia-notification-actions > li:hover {
+    /* remove default background change as due to the different scaling of the items it might look odd */ 
+    background-color: transparent;
+    transform: scale(1.5);
+}
```

Comment:
@martin-fleck-at we should neither remove the background, or change the scaling of the icons when hovering as it is the old way of rendering icons, and does not align with the rest of the icon buttons or toolbar items present in the framework. 

It currently breaks the updates from:
- https://github.com/eclipse-theia/theia/pull/9864
- https://github.com/eclipse-theia/theia/pull/10078